apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/model/tilbakekreving/TilbakekrevingBrevData.kt

```diff
@@ -0,0 +1,127 @@
+                summer = perioderSummert(tilbakekreving),
+            )
+        }
+
+        private fun sjekkOmHarRenter(tilbakekreving: Tilbakekreving): Boolean {
+            val forekomstAvRentetillegg =
+                tilbakekreving.perioder.find { periode ->
+                    periode.ytelse.rentetillegg.let { it != null && it > 0 }
+                }
+            return forekomstAvRentetillegg != null
+        }
+
+        private fun sjekkOmHarForeldet(tilbakekreving: Tilbakekreving): Boolean {
+            val forekomstAvRentetillegg =
+                tilbakekreving.perioder.find { periode ->
+                    periode.ytelse.resultat.let { it != null && it == TilbakekrevingResultat.FORELDET }
+                }
+            return forekomstAvRentetillegg != null
+        }
+
+        private fun tilbakekrevingsPerioder(tilbakekreving: Tilbakekreving) =
+            tilbakekreving.perioder.map { periode ->
+                TilbakekrevingPeriodeData(
+                    maaned = periode.maaned.atDay(1),
+                    beloeper =
+                        periode.ytelse.let { beloeper ->
+                            TilbakekrevingBeloeperData(
+                                feilutbetaling = Kroner(beloeper.beregnetFeilutbetaling ?: 0),
+                                bruttoTilbakekreving = Kroner(beloeper.bruttoTilbakekreving ?: 0),
+                                fradragSkatt = Kroner(beloeper.skatt ?: 0),
+                                nettoTilbakekreving = Kroner(beloeper.nettoTilbakekreving ?: 0),
+                                renteTillegg = Kroner(beloeper.rentetillegg ?: 0),
+                            )
+                        },
+                    resultat =
+                        periode.ytelse.resultat?.name
+                            ?: throw BrevDataTilbakerevingHarManglerException("Alle perioder må ha resultat"),
+                )
+            }
+
+        private fun perioderSummert(tilbakekreving: Tilbakekreving): TilbakekrevingBeloeperData {
+            var sumFeilutbetaling = 0
+            var sumBrutto = 0
+            var sumSkatt = 0
+            var sumNetto = 0
+            var sumRenter = 0
+            tilbakekreving.perioder.forEach {
+                val beloeper = it.ytelse
+                sumFeilutbetaling += beloeper.beregnetFeilutbetaling ?: 0
+                sumBrutto += beloeper.bruttoTilbakekreving ?: 0
+                sumSkatt += beloeper.skatt ?: 0
+                sumNetto += beloeper.nettoTilbakekreving ?: 0
+                sumRenter += beloeper.rentetillegg ?: 0
+            }
+            return TilbakekrevingBeloeperData(
+                feilutbetaling = Kroner(sumFeilutbetaling),
+                bruttoTilbakekreving = Kroner(sumBrutto),
+                fradragSkatt = Kroner(sumSkatt),
+                nettoTilbakekreving = Kroner(sumNetto),
+                renteTillegg = Kroner(sumRenter),
+            )
+        }
+    }
+}
```

Comment:
[.sumOf()](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/sum-of.html)

Reply:
Hm ja. Jeg droppa det for å ikke iterere alle beløper 5 ganger. Men så mange beløper blir det vel ikke så det koster ikke så mye :)